### DISCUSSION TITLE
关于自动播放视频的问题

### DISCUSSION THREAD

**Discussion by 7Azure** [the1812/Bilibili-Evolved#2101] [综合讨论 / General]
我记得之前是需要激活标签页才会自动播放视频，现在新建后台标签页并没激活页面也会自动播放。
我鼠标中键同时后台打开多个视频以便稍后观看，所有的标签全都自动播放声音了，得一个个去暂停。

**Comment by 7Azure** [comment]
就比如视频推荐【接下来播放】，我会浏览一遍列表后挑选多个同时打开供稍后看

**Comment by the1812** [comment]
可是我这没有自动播放的功能(
b站播放器设置里可以关掉自动播放

**Reply by 7Azure** [reply]
可是关闭了你的脚本，就不会后台自动播放了，我用的是Edge，只启用了你的脚本。

**Reply by 7Azure** [reply]
这个问题也是这几天更新后出现的，之前不会这样

**Reply by 7Azure** [reply]
好吧，我关掉自动播放
